tgui/src/interfaces/ntos_supermatter_monitor.ract

```diff
@@ -0,0 +1,70 @@
+
+<script>
+  component.exports = {
+    data: {
+	  integState(charge) {
+	    let max = 100
+	    if(charge == max) return 'good'
+		else if(charge > (max/2)) return 'average'
+		else return 'bad'
+	    }
+      }
+    }
+
+  component.exports = {
```

Comment:
@RemieRichards for future reference, is there a way to do an else if, or switch statement in tgui?